docker/env/version

```diff
@@ -1 +1 @@
-0.35
+0.37
```

Comment:
why not 0.36?

Reply:
cause 0.36 exist in the avocado removal branch... and we have images with that version across our CI servers.

having two PRs changing the this version at the same time causes a bit of a mess.